### PR TITLE
Changed cards for home page

### LINKED ISSUE #73: UI Update/CleanUp for GSoC18
## Description
As a developer,
I need to complete the branding of the app,
so that I can push a final product to production.

## Mocks

-

-

-

-

-

### Update [Required]
- [90%] Navbar updated to white buttons on the top right, with blue on hover, and underline on active.
- [ ] Footer updated to three links on the right: [Code of Conduct](https://ghc.anitab.org/code-of-conduct/), [Terms & Conditions](https://anitab.org/terms-of-use/), and [Privacy Policy](https://anitab.org/privacy-policy/)
- [ ] Footer remove "Contact Us" and the "We connect" statement.
- [ ] Remove blue footer and update to 
image

- [ ] Headers updated to ---
- [ ] Sub-headers updated to ---
- [ ] Text updated to ---
- [ ] Buttons updated to ---

- [ ] Update favicon to [32x32 png](https://anitab.org/wp-content/assets/images/favicons/favicon-32x32.png) on desktop, [16x16 png](https://anitab.org/wp-content/assets/images/favicons/favicon-16x16.png) on mobile/tablet.

### Enhancement to Update [Optional]
- [ ] [LIST ITEMS]

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
8 hours

### PR DESCRIPTION
**Pull request by anumehaagrawal**
### Description
In this PR I have changed cards in the home page to keep it consistent with rest of the page

Fixes #73 
### Type of Change:
**Delete irrelevant options.**

- Code
- Quality Assurance
- User Interface
- Outreach
- Documentation

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
This has been tested by running it on local host

### Checklist:
**Delete irrelevant options.**

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
 

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by anumehaagrawal** [comment]
@Tharangi Resize as in ?

**Comment by Tharangi** [comment]
@anumehaagrawal: Could you check my comment in your previous PR - #81

**Comment by Rupinderwadali** [comment]
@anumehaagrawal I added  a comment in PR #81 for solving the issue of blur images. Have a look.

File: src/app/home/home.component.html [hunk 1]
```diff
@@ -1,35 +1,16 @@
-<mdb-carousel [isControls]="true" class="carousel slide carousel-fade" [animation]="'fade'">
-    <mdb-slide>
-        <div class="view">
-            <img class="d-block w-100" src="https://mdbootstrap.com/img/Photos/Slides/img%20(68).jpg" alt="First slide">
```
Comment (Rupinderwadali) [inline src/app/home/home.component.html]: Using the div instead of image or use height equals to vh helps for proper view of slider at the mobile devices

Comment (anumehaagrawal) [reply src/app/home/home.component.html]: The vh thing worked :) Should I use div also ?

**Pull request closed without merging**